src/graphics/program-lib/chunks/cubeMapRotate.frag

```diff
@@ -0,0 +1,19 @@
+#ifdef CUBEMAPROT
+uniform mat3 cubeMapRotationMatrix;
+#endif
+
+vec3 cubeMapRotate0(vec3 refDir) {
+#ifdef CUBEMAPROT
+    return refDir*cubeMapRotationMatrix;
+#else
+    return refDir*vec3(-1,1,1);
+#endif
+}
+
+vec3 cubeMapRotate1(vec3 refDir) {
+#ifdef CUBEMAPROT
+    return (refDir*cubeMapRotationMatrix)*vec3(-1,1,1);
+#else
+    return refDir;
+#endif
+}
```

Comment:
Remember to stick to our coding style:
* space after comma
* spaces around operators
🤓

Reply:
falling foul of my reliance on lint - I'll do a spacing pass on all the shaders chunks I touched

Reply:
shader chunk spacing should all be fixed now - thanks!